tests/format/mdx/mdx/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -316,6 +324,14 @@ semi: false
 | -------- | ------------------- |
 | Text     | <Hello>Text</Hello> |
 
+---
+
+A
+
+{/* JS-style comment */} comment.
```

Comment:
Anyone any clue why there’s a blank line inserted before the comment (but not after?). Ideally there would be no blank lines

Reply:
Seems inserted [here](https://github.com/prettier/prettier/blob/b2fa604e77dab3c9e33e832f639703e7d22f128f/src/language-markdown/printer-markdown.js#L719)

Reply:
thanks @fisker, fixed!